lambda/models/lambda_functions.py

```diff
@@ -63,6 +65,14 @@ async def model_not_found_handler(request: Request, exc: ModelNotFoundError) ->
     return JSONResponse(status_code=404, content={"message": str(exc)})
 
 
+@app.exception_handler(RequestValidationError)  # type: ignore
+async def validation_exception_handler(request: Request, exc: RequestValidationError):
+    """Handle exception when request fails validation and and translate to a 422 error."""
+    return JSONResponse(
```

Comment:
Do we know that the only exceptions that will be caught by this are 422 errors? There aren't in lingering issues that could throw runtime exceptions (500 errors) that could be caught by this and accidentally translated to 422s?

Reply:
By default FastAPI [already returns](https://github.com/fastapi/fastapi/blob/master/fastapi/exception_handlers.py#L20-L26) a [422](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422) for `RequestValidationError`…which seems appropriate. I wouldn't expect anything that is supposed to be a 5xx to be throwing a `RequestValidationError`. This is really re-implementing the default functionality for clarity by having all our exception handling in one place. I'd hope to start adding "type" to all the error responses so it is easier to discriminate between them in the future easily on the front end.